site/assets/js/map.js

```diff
@@ -72,3 +74,7 @@ var initMap = function () {
 };
 
 initMap();
+window.addEventListener("resize", function(){
```

Comment:
nice change. I got rid of the animation (same branch), b/c that looked super weird.
we might want to debounce this listener? otherwise the resize event fires super often...